.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: run
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+jobs:
+  tests:
+    name: unit-tests
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+    
+    - name: Run Era Test Node
+      uses: dutterbutter/era-test-node-action@latest
+
+    - name: Install Dependencies
+      run: yarn install
+    
+    - name: Run Tests
+      run: |
+        yarn ci:tests
```

Comment:
suggestion: We should consider always printing out the logs of `era_test_node` in case the tests fail

```
- name: Era Test Node Logs
  if: always()
  run: cat ./era_test_node.log
```

Reply:
I think if a job fails it would be best the user updates the action to use logs:debug and then make use of the log file which I don't think should be used by default?